gatoryaml/gator_yaml.py

```diff
@@ -4,8 +4,8 @@
 
 def dump(header, body, indent=4, spaces=4) -> str:
     """Input dictionary is parsed. returns string of valid YAML"""
-
-    output = parse_header(header)
+    output = "---\n"
+    output += parse_header(header)
```

Comment:
Personally I feel like the `---` should be inside `parse_header`, but it doesn't matter.